tests/actions/vstudio/vc2010/test_link.lua

```diff
@@ -460,3 +460,20 @@
 </Link>
 		]]
 	end
+
+--
+-- Test ignoring default libraries.
+--
+
+	function suite.ignoreDefaultLibraries()
+		ignoreDefaultLibraries { "lib1.lib", "lib2.lib" }

```

Comment:
This makes me wonder if should support lib names without `.lib` by auto-appending like the last issue?